torch/_dynamo/variables/user_defined.py

```diff
@@ -306,6 +307,12 @@ def var_getattr(self, tx, name):
             return variables.UserMethodVariable(
                 subobj.fget, self, **options
             ).call_function(tx, [], {})
+        elif isinstance(subobj, staticmethod):
+            return variables.UserFunctionVariable(subobj.__get__(self.value), **options)
+        elif isinstance(subobj, classmethod):
+            return variables.UserMethodVariable(subobj.__func__, self, **options)
+        elif isinstance(subobj, types.FunctionType):
```

Comment:
This line is the real fix, the other lines are to consolidate these scenarios to make code more clear.